### ISSUE TITLE
Missing 'this.' error doesn't trigger other quick fixes

### ISSUE THREAD

**Issue by DanielRosenwasser** [Microsoft/TypeScript#25846]
Found by @wycats

```ts
// ./bar.ts
export function blah() {
}
```

```ts
// ./foo.ts
let blahh = 10;

export class C {
    blah() {
    }

    bar() {
        blah
    }
}
```

Try to request quick fixes on `blah`. You'll only get a `this.` quick fix suggestion, however, we should also provide

1. a suggestion to fix the spelling to `blahh`
1. a suggestion for the import `blah`

We should consider making a central list for all name resolution errors so that we suggest quick fixes appropriately.
